assets/js/components/ExecutionResults/CheckResultDetail/FactValue.jsx

```diff
@@ -6,7 +6,7 @@ function FactValue({ className, data }) {
   return typeof data === 'object' ? (
     <ObjectTree className={className} data={data} />
   ) : (
-    <span className={className}>{data}</span>
+    <span className={className}>{`${data}`}</span>
```

Comment:
```suggestion
    <span className={className}>{JSON.stringify(data)}</span>
```

Reply:
unfortunately that outputs quoted strings like `"foobar"` instead of simple `foobar`.
Any preferences?

Reply:
ç_ç